akash1/HTML/ASSIn1/assignment1.Q2.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Q2</title>
+</head>
+<body>
+    <h1>HTML</h1>
+    <h3>INTRODUCTION</h3>
+    HTML is a standred markup language for Web Pages.
+    <h1>HTML Editor
+        <h3>Notepad or Textedit</h3>
+        Web pages can be created and modified by using professional HTML editors.<br>
+        However, for learning HTML we recommend a simple text editor like Notepad(PC)or TextEdit(Mac).
+</body>
```

Comment:
do proper indentation as </html> is a parent div and <head> and <body> tag are child of <html> tag u should give four tab space for child element from parent tag so make this change in all files.